docs/src/routes/docs/components/star-rating.svx

```diff
@@ -0,0 +1,91 @@
+---
+name: StarRating
+---
+
+<script>
+  import { StarRating } from 'attractions';
+  import { ThumbsUpIcon } from 'svelte-feather-icons';
+  import Showcase from 'src/containers/docs/showcase.svelte';
+  import ColorPreview from 'src/components/docs/color-preview.svelte';
+
+  let restaurantRating = 1;
+</script>
+
+# StarRating
+
+An input element used for rating with a number of stars.
+
+<Showcase>
+  <div slot="showcase" class="padded">
+    <StarRating name="restaurant" bind:value={restaurantRating} />
+    <br>
+    <StarRating name="hotel" value={5} max={7} disabled>
+      <span slot="icon"><ThumbsUpIcon size="24" /></span>
+    </StarRating>
+  </div>
+  <div slot="source">
+
+```svelte
+<script>
+  let restaurantRating = 1;
+</script>
+
+<StarRating name="restaurant" bind:value={restaurantRating} />
+<br>
+<StarRating name="hotel" value={5} max={7} disabled>
+  <span slot="icon"><ThumbsUpIcon size="24" /></span>
+</StarRating>
+```
+
+  </div>
+</Showcase>
+
+
+## Properties {#properties}
+
+### Functional Properties
+
+| Name | Default | Type | Description |
+| ---- | ------- | ---- | ----------- |
+| ==**`name`**==&nbsp;<span class="required">*</span> |  | `string` | The name to assign to all stars belonging to the same group. Check [MDN docs](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input#htmlattrdefname) for more information. |
+| **`max`** | `5` | `number` | How many stars to be displayed. |
+| **`value`** | `null` | `number` | Current amount of selected stars. |
+| **`disabled`** | `false` | `boolean` | Disables star selection. |
+| **`events`** | `[]` | `Array` | List of handlers for the [Events action](): `[{ name, handler }]`. |
```

Comment:
We should probably mention what element are these handlers attached to

Reply:
I just copied it from the button exactly as it is (including the empty link).